src/lib/Scenes/Artwork/Components/CommercialPartnerInformation.tsx

```diff
@@ -20,26 +25,31 @@ export class CommercialPartnerInformation extends React.Component<Props> {
           <>
             <Spacer mb={1} />
             <Sans size="3t" color="black60">
-              {availabilityDisplayText} {artwork.partner.name}
-            </Sans>
-            {artworkEcommerceAvailable &&
-              !!artwork.shippingOrigin && (
-                <Sans size="3t" color="black60">
-                  Ships from {artwork.shippingOrigin}
-                </Sans>
-              )}
-            {artworkEcommerceAvailable &&
-              !!artwork.shippingInfo && (
-                <Sans size="3t" color="black60">
-                  {artwork.shippingInfo}
-                </Sans>
-              )}
-            {artworkEcommerceAvailable &&
-              !!artwork.priceIncludesTaxDisplay && (
-                <Sans size="3t" color="black60">
-                  {artwork.priceIncludesTaxDisplay}
-                </Sans>
+              {artwork.sale ? (
```

Comment:
Will this also show up for auction works? That doesn't seem ideal given we have other treatment for auction sales.

It would be worth adding a comment or logic so this _only_ works if the sale is not an auction.

Also, have you verified what the app's treatment of non-auction sales is? I'm not sure if it was designed to work for that use-case.

Reply:
Good point. I believe we're not linking because the sale page doesn't look right for these. It shows number of bids, for example.

![Image from iOS](https://user-images.githubusercontent.com/19369/71922267-a39bc800-3158-11ea-8146-caf60512c486.png)